qbft/spectest/tests/messages_spectest.go

```diff
@@ -25,7 +25,7 @@ func (test *MsgSpecTest) Run(t *testing.T) {
 			continue
 		}
 
-		if msg.Message.MsgType == qbft.RoundChangeMsgType && len(msg.Message.RoundChangeJustification) == 0 {
+		if msg.Message.RoundChangePrepared() && len(msg.Message.RoundChangeJustification) == 0 {
```

Comment:
what did you try to do here?

Reply:
It's for `messages.RoundChangeDataInvalidJustifications` (https://github.com/bloxapp/ssv-spec/blob/ssz-fix-remaining-tests-issues/qbft/spectest/tests/messages/rc_invalid_justifications.go#L11). 

We used to have this check in https://github.com/bloxapp/ssv-spec/blob/main/qbft/messages.go#L140 but now we don't have it when validating a message (https://github.com/bloxapp/ssv-spec/blob/ssz-fix-remaining-tests-issues/qbft/messages.go#L122). 

I think I need to remove this check because it's in a test runner and doesn't actually test what we need to test.

Should we add a justification length check of prepared messages in https://github.com/bloxapp/ssv-spec/blob/ssz-fix-remaining-tests-issues/qbft/messages.go#L122 or should we just remove this check and remove the `messages.RoundChangeDataInvalidJustifications` test?

Reply:
mm not sure what it's for...  I think we can remove it

Reply:
Removed